pkg/clients/etcd.go

```diff
@@ -0,0 +1,40 @@
+package clients
+
+import (
+	"fmt"
+	"time"
+
+	logging "github.com/op/go-logging"
+
+	"github.com/coreos/etcd/client"
+)
+
+type EtcdConfig struct {
```

Comment:
@eriknelson +1 to `EtcdConfig` remaining `Config`. I just added that as a comment to my review.